contribution/testing_guide.md

```diff
@@ -56,46 +56,15 @@ $ kubectl -n [namespace name] exec -it [pod name] -- bash -c [command]
 
 ## 1.6. Check generated alerts
 
-- Log file
+- Log using [karmor](https://github.com/kubearmor/kubearmor-client) cli tool
 
     ```text
-    $ tail (-f) /tmp/kubearmor.log
-    ```
-
-    If you changed the location of a log file, replace the default file path to your file.
+    $ karmor log (options...)
 
-    ```text
-    $ tail (-f) [your log file path]
+    For example: 
+    $ karmor log --json
```

Comment:
```suggestion
    $ karmor log [options...]
```

Followed up by available options, like the earlier log client options were explained. Not everyone wants to see JSON :)